composables/useForecastData.ts

```diff
@@ -0,0 +1,61 @@
+// Takes a ukranian oblast and scales it by the value provided
+function formatAndScaleValue(value, oblastNameUkrainian, oblastScales) {
+  
+  // Get the proper scale for this oblast name
+  const scale = oblastScales ? oblastScales[oblastNameUkrainian] : 1;
+
+  // If the scale is undefined, set it to 1
+  const sliderScale = (scale >= 0 ? scale : 1);
+
+  // Return the value scaled by the sliderScale
+  return formatValue(value * sliderScale);
+}
+
+// Takes a number and rounds it so it's not long
+function formatValue(value) {
+  return (+value).toFixed(1);
+}
```

Comment:
are we able to import these from `helpers.js`

Reply:
in a mildly distressing turn of events... the app gets a lot slower if these are imported from helpers for some reason, but I left the code commented out explaining this, we can come back to it... sorry